server/events.go

```diff
@@ -691,6 +691,7 @@ func (s *Server) sendStatsz(subj string) {
 			ni := v.(nodeInfo)
 			ni.stats = jStat.Stats
 			ni.cfg = jStat.Config
+			ni.tags = s.getOpts().Tags
```

Comment:
I think it would be better to make copies of the options's tags value here. Since Tags is a type but an alias to []string, you could use `copyStrings(s.getOpts().Tags)`

Reply:
Will do.